pages/about.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About us</title>
+  </head>
+  <body>
+    <section class="hero">
+      <img
+        src="/images/logo.png (2).png"
+        alt="Seven Foods And Grill Logo"
+        width="150"
+        height="auto"
+      />
+      <nav class="navbar">
+        <ul>
+          <li><a href="./pages/about.html">About</a></li>
+          <li><a href="../pages/contact.html">Contact</a></li>
+        </ul>
+        <hr />
+      </nav>
```

Comment:
The navigation links here are pointing to ./pages/about.html and ../pages/contact.html. Ensure that these paths are accurate and will lead to the intended pages.